app/src/main/java/de/westnordost/streetcomplete/quests/shop_type/CheckShopExistence.kt

```diff
@@ -0,0 +1,60 @@
+package de.westnordost.streetcomplete.quests.shop_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.elementfilter.toElementFilterExpression
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmElementQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.IS_SHOP_EXPRESSION
+import de.westnordost.streetcomplete.osm.IS_SHOP_OR_DISUSED_SHOP_EXPRESSION
+import de.westnordost.streetcomplete.osm.LAST_CHECK_DATE_KEYS
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.osm.isShopExpressionFragment
+import de.westnordost.streetcomplete.osm.updateCheckDate
+import de.westnordost.streetcomplete.quests.existence.CheckShopExistenceForm
+
+class CheckShopExistence : OsmFilterQuestType<Unit>() {
+
+    // opening hours quest acts as a de facto checker of shop existence, but some people disabled it.
+    // separate from CheckExistence as very old shop with opening hours should show
+    // opening hours resurvey quest rather than this one (which would cause edit date to be changed
+    // and silence all resurvey quests)
+
+    override val elementFilter = """
+        nodes with (
+          (${isShopExpressionFragment()})
```

Comment:
Are you sure that the extra `(...)` are necessary here? In the code it looks like that function already adds `(...)`

Reply:
added for safety but actually seems not needed - removed